Sources/OTel/Metrics/OTelMetricProducer/OTLPMetricsDataModel.swift

```diff
@@ -132,9 +121,8 @@ public struct OTelHistogramDataPoint: Equatable, Sendable {
     public var exemplars: [OTelExemplar]
 }
 
-@_spi(Metrics)
 public struct OTelExemplar: Equatable, Sendable {
-    var spanID: OTelSpanID?
-    var observationTimeNanosecondsSinceEpoch: UInt64
-    var filteredAttributes: [OTelAttribute]
+    public var spanID: OTelSpanID?
```

Comment:
Do we envision end users mutating this? Might be good to keep it `package(set)` if not.